src/ResourceManager/Network/Commands.Network/Models/PSBackendAddressPool.cs

```diff
@@ -21,10 +21,13 @@ namespace Microsoft.Azure.Commands.Network.Models
 
     public class PSBackendAddressPool : PSChildResource
     {
+        [JsonProperty(Order = 1)]
```

Comment:
How does making every property Order=1 change the behavior?  Do you have a test for the new serialization?